qa-tests-backend/src/test/groovy/ProcessesListeningOnPortsTest.groovy

```diff
@@ -228,6 +238,96 @@ class ProcessesListeningOnPortsTest extends BaseSpecification {
         assert list3.size() == 0
     }
 
+    @Tag("BAT")
+    def "Verify networking endpoints disappear when process is terminated"() {
+        given:
+        "Two deployments that listen on ports are started up"
+
+        rebuildForRetries()
+        def clusterId = ClusterService.getClusterId()
+
+        String deploymentId3 = targetDeployments[2].getDeploymentUid()
+
+        def gotCorrectNumElements = waitForResponseToHaveNumElements(1, deploymentId3, 240)
```

Comment:
First make sure that we see the listening endpoint in the API when it is open. Add a comment here.